### PR TITLE
Remove ECS SG egress and revert to EDGE API Gateway type

### PR DESCRIPTION
**Pull request by tommyb82**
Unfortunately required a targeted destroy of the API gateway REST API resource first - cannot change from "PRIVATE" to "EDGE" 😞 

Will need to run the same for infra-services-fat and on TST and PPD for both prior to deployment then update CCS web team with new URL and Postman environments..

```
terraform destroy -target module.deploy.module.api.aws_api_gateway_rest_api.scale
```

### CODE REVIEW

File: terraform/modules/api/main.tf [hunk 1]
```diff
@@ -69,6 +69,10 @@ resource "aws_api_gateway_rest_api" "scale" {
   name        = "SCALE:EU2:${upper(var.environment)}:API:Shared"
   description = "SCALE API Gateway"
 
+  endpoint_configuration {
```
Comment (adrianmilne) [inline terraform/modules/api/main.tf:72]: This looks like a new addition (compared to the pre-change version) - but guess it has no real impact - just checking

**Review by adrianmilne — APPROVED** [review]
👍  Nice one - I tried to apply and got the expected error. Ran your targeted destroy and reapplied and all worked fine (required 2 attempts to destroy and recreate due to the concurrent modification known issue). Tested via Postman.

File: terraform/modules/api/main.tf (hunk 1 above)
Comment (tommyb82) [reply terraform/modules/api/main.tf:72]: Can remove it but makes the type explicit which seems like a sensible thing given the circumstances.. don't mind 🤔

**Comment by tommyb82** [comment]
> 👍 Nice one - I tried to apply and got the expected error. Ran your targeted destroy and reapplied and all worked fine (required 2 attempts to destroy and recreate due to the concurrent modification known issue). Tested via Postman.

Cool, cheers - just doing the equivalent for FaT now.  Then you'll need to update the shared infra from `develop` to remove the VPC endpoint as the final step - I've reverted the develop branch there already.

**Pull request merged**